lib/owdit.js

```diff
@@ -2,9 +2,46 @@ const exec = require("executive");
 const formatter = require("./formatters/default")
 const Client = require("node-rest-client").Client;
 const util = require("util");
+const fs = require("fs");
+const path = require("path");
+const rcFilename = ".owditrc";
+
+// Read from .owditrc the lists of packages that should be excluded or only generate warnings
+function readWarnsAndExcludes(cwd) {
+  let warns = [];
+  let excludes = [];
+  const rcFile = path.resolve(cwd, rcFilename);
+
+  if (fs.existsSync(rcFile)) {
+    try {
+      const content = JSON.parse(fs.readFileSync(rcFile));
+
+      warns = content.warns || [];
+      excludes = content.excludes || [];
+    } catch(err) {
+      // Display any error occurred when reading or parsing .owditrc but don't stop the check.
+      // At this moment, this file's sole purpose is to make the vulnerability check less strict.
+      // Going on when there has been an error at this step means all vulnerabilities will be taken
+      // into account and the check can only get stricter.
+      console.log(err);
+      console.log("Error reading or parsing .owditrc. All dependencies will be scanned.");
+    }
+  }
+
+  if (warns.length) {
+    console.log(`Vulnerabilities in these packages will only generate warnings: ${warns.join(", ")}.`)
+  }
+  if (excludes.length) {
+    console.log(`Vulnerabilities in these packages will be IGNORED: ${excludes.join(", ")}.`)
+  }
+
+  return {warns, excludes};
+}
 
 module.exports.check = function(cwd, callback)
 {
+  const {warns, excludes} = readWarnsAndExcludes(cwd);
```

Comment:
I will look into how I can adapt the docs or change the code.

When run as a command, owdit actually does a `npm ls` inside the current working directory, and npm is smart enough to look recursively up in the directory tree for a `package.json` when one is absent from the cwd. So what I documented is erroneous: `.owditrc` will only be taken into consideration when present in the cwd.